pom.xml

```diff
@@ -67,7 +67,7 @@
         <geronimo-javamail.version>1.8.4</geronimo-javamail.version>
         <commons-codec.version>1.13</commons-codec.version>
         <commons-logging.version>1.2</commons-logging.version>
-        <apache-mailet.version>3.1.0-SNAPSHOT</apache-mailet.version>
+        <apache-mailet.version>3.4.0-SNAPSHOT</apache-mailet.version>
```

Comment:
Why is that version needed at all?

The JDKIM mailet was moved to James code, all related POM management / code should be removed here as well.